### ISSUE TITLE
Send transaction uses wrong account

### ISSUE THREAD

**Issue by bas-vk** [SilentCicero/meteor-dapp-cosmo#11]
I have 2 accounts loaded in Cosmo. When I send a transaction to the node with the second account selected Cosmo still uses the first account for the request that is send to the node.

**Comment by ice09** [comment]
Same here. Looks like the first account is used always, even if "from" is selected to account with index >0.
However, your product is gorgeous and was my entry door to ethereum, thanks a lot!

**Comment by SilentCicero** [comment]
Yeah, Cosmo needs a big upgrade. Please feel free to contribute =D

**Comment by ice09** [comment]
Ok, ist there a good starting point? A simple "start" for developers would be great, would this be possible (esp. the "public" part of the Meteor app I don't really understand)?

**Comment by nitinjaiin** [comment]
i have a solidity contract with input parameter of 'address' type, when i invoke this contract using sendTransaction function from my meteor dapp and pass the account address it doesn't work but testing the same contract using cosmo editor it works.
do I need to pass the account address in a specific way from the meteor app?
